src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -608,6 +608,29 @@ public static IEnumerable<object> GetAsync_Chunked_VaryingSizeChunks_ReceivedCor
                 });
             });
         }
+
+        [Theory]
+        [InlineData("get", "GET")]
+        [InlineData("head", "HEAD")]
+        [InlineData("post", "POST")]
+        [InlineData("put", "PUT")]
+        [InlineData("other", "other")]
+        [InlineData("SometHING", "SometHING")]
+        public async Task CustomMethod_SentUppercasedIfKnown(string specifiedMethod, string expectedMethod)
```

Comment:
To be complete, you should probably list all 9 of the methods you added to the dictionary.

Reply:
This is the base test that also runs on netfx and WinHttpHandler, which only agree on these few.  There's a derived test specific to SocketsHttpHandler which adds the rest, except for CONNECT, which is special and can't be tested in this fashion, and also isn't meant to be supplied by a user.

Reply:
ok